tox.ini

```diff
@@ -18,12 +18,8 @@ setenv =
     PYTHONPATH = {toxinidir}/mock_apps
     DJANGO_SETTINGS_MODULE = test_settings
 deps =
-    django111: Django>=1.11,<2
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
     django22: Django>=2.2,<2.3
-    drf37: djangorestframework<3.8.0
-    drf38: djangorestframework<3.9.0
+    django30: Django>=3.0,<3.1
     drf39: djangorestframework<4.0.0
```

Comment:
Sorry, one last thing I just spotted; this actually gets DRF 3.11.0, we want `djangorestframework<3.10.0` instead to get DRF 3.9.4.

Reply:
Updated as per suggestion.